docs/operator-docs/cmr-operations.md

```diff
@@ -4,14 +4,18 @@ title: CMR Operations
 hide_title: false
 ---
 
-This document will outline basic procedures to interact with CMR on a per-granule basis.
+This document will outline basic procedures to interact with CMR on a per-granule basis, both via the Cumulus Dashboard, and Cumulus API requests.
```

Comment:
Nit:  I'm wondering if we should include something here regarding incompatible CMR metadata given the issues NSIDC had with ISO xml?

Reply:
Hrm. I was going to say something about PostToCmr being format agnostic but I'm not quite sure that's true.